user/templates/profile/profile-beta.html

```
@@ -11,15 +11,196 @@
+                        <button id="toggle-contact-form"
+                            class="text-sm text-blue-600 hover:underline dark:text-blue-500">
+                            {% translate "Add Contact" %}
+                        </button>
+                        -->
+                    </div>
+                    <ol class="w-full mt-2 md:mt-12 space-y-2">
+                        {% for contact in contact_profiles %}
+                            <li class=" dark:md:border-gray-700/30 md:hover:bg-blue-200 dark:md:hover:bg-gray-700/30 rounded-lg p-2 duration-300 group">
+                                <div class="flex justify-between items-center">
+                                    {% if contact.platform.icon %}
+                                        <a href="{% if contact.platform.name == "Telephone" %} tel:{{contact.username}}{% else %}mailto:{{contact.username}}{% endif %}" class="w-8 h-8 flex items-center gap-2">
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)